src/main/java/edu/kis/powp/command/ComplexCommand.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.command;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand{
+
+	private ArrayList<DriverCommand> commands = new ArrayList<>();
```

Comment:
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.